src/generate/apiCollection.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ApiGroup } from "./apiGroup";
+
+/**
+ * A collection of multiple groups of APIs stored by key.
+ */
+export class ApiCollection {
```

Comment:
One thing to just keep in mind...we group the APIs in the sdk we've built because of the sheer number....but we should definitely support a flat structure as well at some point.

Reply:
Absolutely. I don't see us making anyone use this class if they don't need it; they can just use a single instance of ApiGroup.